yb-voyager/src/reporter/stats/streamImportReporter.go

```diff
@@ -121,6 +121,6 @@ func (s *StreamImportStatsReporter) BatchImported(numInserts, numUpdates, numDel
 }
 
 func (s *StreamImportStatsReporter) getIngestionRateForLastNMinutes(n int64) int64 {
-	window := 2*n + 1 //2*n as sliding window every 30 secs
+	window := 6*n + 1 //6*n as sliding window every 10 secs
 	return lo.Sum(s.eventsPer30Secs[1:window]) / n
```

Comment:
Now the `eventsPer30Secs` name is misleading ;-) . How about naming it as just `slidingWindow`?

Reply:
I also just realized that. will rename.